torch/testing/_creation.py

```diff
@@ -139,7 +142,12 @@ def clamp(a, l, h):
         raise ValueError("make_tensor: requires_grad must be False for integral dtype")
 
     if dtype is torch.bool:
-        result = torch.randint(0, 2, shape, device=device, dtype=dtype)  # type: ignore[call-overload]
+        ranges = (0, 2)
```

Comment:
Yes, this looks weird, but is on par with the other branches. The next commit in this stack cleans this up.

Reply:
OK